integration-tests/src/test/java/oracle/weblogic/kubernetes/assertions/impl/Kubernetes.java

```diff
@@ -487,13 +487,18 @@ public static V1Pod getPod(String namespace, String labelSelector, String podNam
             null, // Timeout for the list/watch call.
             Boolean.FALSE // Watch for changes to the described resources.
         );
+    getLogger().info("REG-> found {0} pods in namespace {1}", v1PodList.getItems().size(), namespace);
```

Comment:
Should `REG` (seems like initials) be something like `DEBUG`?

Reply:
No, they _are_ my initials, and are supposed to remind me to take that line out before putting it in a PR. Oops.